modules/mapping-utils/package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "@kfarranger/middleware": "1.5.2",
     "babel-polyfill": "^6.26.0",
-    "graphql-fields": "^1.0.2",
+    "graphql-fields": "^2.0.3",
```

Comment:
did you check in the release notes what are the breaking changes?

Reply:
there does not seem to be none.